src/redis/BetterSocialCronQueue.js

```diff
@@ -59,17 +59,17 @@ class BetterSocialCronQueue extends BetterSocialQueue {
   static process(job, done, queueInjection) {
     const {flag} = job.data;
     console.log(`============= DOING CRON JOB (${flag})  =============`);
-    if (flag === 'dailyCredderUpdate') {
-      return credderDailyScoreProcess(job, done, queueInjection?.credderScoreQueue);
-    }
+    // if (flag === 'dailyCredderUpdate') {
+    //   return credderDailyScoreProcess(job, done, queueInjection?.credderScoreQueue);
+    // }
 
     if (flag === 'dailyDeleteExpiredPost') {
       return deleteExpiredPostProcess(job, done, queueInjection?.removeActivityQueue);
     }
 
-    if (flag === 'dailyRssUpdate') {
-      return rssProcess(job, done);
-    }
+    // if (flag === 'dailyRssUpdate') {
+    //   return rssProcess(job, done);
+    // }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similarly, the code related to 'dailyRssUpdate' has also been commented out. If this functionality is not required, consider removing these lines of code.

```diff
- // if (flag === 'dailyRssUpdate') {
- //   return rssProcess(job, done);
- // }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->